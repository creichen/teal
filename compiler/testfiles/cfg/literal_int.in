fun g() = {
  return 2;
}
