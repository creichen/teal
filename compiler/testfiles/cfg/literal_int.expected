Entry#0 @ 1 :
  -> IntConstant#0
ReturnStmt#0 @ 2 :
  -> Exit#0
IntConstant#0 @ 2 :
  -> ReturnStmt#0
Exit#0 @ 3 :
